tests/e2e/testcases/icm/sendMsg/suite.go

```diff
@@ -0,0 +1,522 @@
+package sendmsg
+
+import (
+	"fmt"
+
+	"github.com/ava-labs/avalanche-cli/tests/e2e/commands"
+	"github.com/ava-labs/avalanche-cli/tests/e2e/utils"
+	ginkgo "github.com/onsi/ginkgo/v2"
+	"github.com/onsi/gomega"
+)
+
+const (
+	ewoqKeyName = "ewoq"
+	subnetName  = "testSubnet"
+	subnet2Name = "testSubnet2"
+	cChain      = "cchain"
+	message     = "Hello World"
+)
+
+var _ = ginkgo.Describe("[ICM] sendMsg", func() {
+	ginkgo.BeforeEach(func() {
+		commands.StartNetwork()
+	})
+
+	ginkgo.AfterEach(func() {
+		commands.CleanNetwork()
+		err := utils.DeleteConfigs(subnetName)
+		gomega.Expect(err).Should(gomega.BeNil())
+		utils.DeleteCustomBinary(subnetName)
+
+		err = utils.DeleteConfigs(subnet2Name)
+		gomega.Expect(err).Should(gomega.BeNil())
+		utils.DeleteCustomBinary(subnet2Name)
+	})
+
```

Comment:
seems that all these tests are for non sov blockchain -> (subnets, not L1s), which is a good start. 

Can we have tests for L1s too? 

This means: 
- CreateSubnetEvmConfigSOV
 - DeploySubnetLocallySOV -> this one should have icm already deployed automatically unless u disable it with a flag

Reply:
I ignore if we want to use this specialized functions, or more generic ones with appropriate args. I tend to prefer the last option as it is in the way of less funtions

Reply:
Do we want to do tests for SOV blockchains now or do we want to just add tests for non SOV blockchains to cover more commands?

Reply:
We want to switch to using SOV , non SOV is planned for deprecation